source/backend/dal/Repositories/AcquisitionFileRepository.cs

```diff
@@ -25,14 +25,41 @@ public class AcquisitionFileRepository : BaseRepository<PimsAcquisitionFile>, IA
         /// <param name="dbContext"></param>
         /// <param name="user"></param>
         /// <param name="logger"></param>
-        public AcquisitionFileRepository(PimsContext dbContext, ClaimsPrincipal user, ILogger<AcquisitionFileRepository> logger, IMapper mapper)
+        public AcquisitionFileRepository(PimsContext dbContext, ClaimsPrincipal user, ILogger<AcquisitionFileRepository> logger, ISequenceRepository sequenceRepository)
             : base(dbContext, user, logger)
         {
+            _sequenceRepository = sequenceRepository;
         }
         #endregion
 
         #region Methods
 
+        /// <summary>
+        /// Generates a new Acquisition Number in the following format.
+        /// </summary>
+        /// <remarks>
+        /// <list type="bullet">
+        /// <item>
+        /// Format: YY-XXXXXXXXXXXX-ZZ
+        /// </item>
+        /// <item>
+        /// Prefix - (YY above) The prefix numbers for an Acquisition file correspond with the MoTI region
+        /// </item>
+        /// <item>
+        /// File # - (XXXXX... above) Acquisition File number is created and each file number should increase in increments of 1.
+        /// The digit base number is unique to the file. Do not pad the number with zeros.
+        /// </item>
+        /// <item>
+        /// Suffix - (ZZ above) The suffix numbers for an Acquisition file defaults to 01.
+        /// </item>
+        /// </list>
+        /// </remarks>
+        /// <returns>The formatted Acquisition File Number.</returns>
+        public static string GenerateAcquisitionNumber(short prefix, long fileNumber, short suffix = 1)
```

Comment:
interesting, I take it that we don't have a method of creating these with anything other then a suffix of 1 for the time being?

Reply:
@devinleighsmith - yes as per the business rules listed in the method comments they want to use "01" as suffix for all acquisition files (at this time). I just future-proofed the method a bit.  If they want to change that in the future, it supports other suffices 

@FuriousLlama I'm fine making it private since no one is calling it outside of this class